### PR TITLE
ADA Due Date Extensions documentation V1

### PR DESCRIPTION
**Pull request by stroilova**
## [DOC-XXXX](https://openedx.atlassian.net/browse/DOC-XXXX)

Documentation for ADA

### Date Needed (optional)

If the release date of a feature is known or estimated, provide it to give reviewers guidance on turnaround time.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: 
- [ ] Subject matter expert: @davestgermain 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [x] Product review: @stroilova 
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/shared/student_progress/course_grades.rst [hunk 1]
```diff
@@ -516,6 +516,38 @@ point scores from ungraded sections are called "Practice Scores".
...
+
+************************************************
+Grant Due Date Extensions for a Specific Learner
+************************************************
+
+You can extend graded subsection due dates for individual learners.
+
+This can be used to support special circumstances, personal emergencies, and disabilities accommodations. 
+
+Instructions
```
Comment (matthugs) [inline en_us/shared/student_progress/course_grades.rst:532]:
```suggestion
Instructions
============
```

would make this a subsection heading, which seems more like what you're intending here. See http://docutils.sourceforge.net/docs/user/rst/quickstart.html#sections for more information on the formatting

File: en_us/shared/student_progress/course_grades.rst [hunk 2]
```diff
@@ -516,6 +516,38 @@ point scores from ungraded sections are called "Practice Scores".
...
+1. Look up learner by email address or username
+2. Choose the graded subsection from the dropdown
+3. Enter new due date 
+4. Specify the reason for the extension
+5. Click **Change due date for student**
+
+
+You can also use this tab to **View** all deadlines or **Reset** a deadline.
+
+What courses is this feature available for?
```
Comment (matthugs) [inline en_us/shared/student_progress/course_grades.rst:546]: You may want to make this more explicitly visually distinct from the text which answers the question.

File: en_us/shared/student_progress/course_grades.rst [hunk 3]
```diff
@@ -516,6 +516,38 @@ point scores from ungraded sections are called "Practice Scores".
...
+To extend a deadline for a learner, visit the **Extensions** tab on the **Instructor Dashboard**
+
+1. Look up learner by email address or username
+2. Choose the graded subsection from the dropdown
+3. Enter new due date 
+4. Specify the reason for the extension
+5. Click **Change due date for student**
+
+
+You can also use this tab to **View** all deadlines or **Reset** a deadline.
```
Comment (matthugs) [inline en_us/shared/student_progress/course_grades.rst:544]: why are "View" and "Reset" capitalized here?

File: en_us/shared/student_progress/course_grades.rst [hunk 4]
```diff
@@ -516,6 +516,38 @@ point scores from ungraded sections are called "Practice Scores".
...
+
+What courses is this feature available for?
+
+All new courses. If you need this feature for an older course, please contact your PM.
```
Comment (matthugs) [inline en_us/shared/student_progress/course_grades.rst]: I would recommend against abbreviating "PM" in this context; it invites questions from anal people who object to undefined abbreviations in technical writing ;)

**Review by matthugs — CHANGES_REQUESTED** [review]

File: en_us/shared/student_progress/course_grades.rst (hunk 3 above)
Comment (stroilova) [reply en_us/shared/student_progress/course_grades.rst:544]: This was mostly to visually draw attention to the other two functionalities in the Exceptions dashboard. We can edit later to add more info if needed - trying to get something quick out, as a preliminary reference.

Comment (matthugs) [reply en_us/shared/student_progress/course_grades.rst:544]: Are they text used in the interface? Are they capitalized there?

**Comment by stroilova** [comment]
Good question - there are two more sections:
1) a section called "Viewing granted extensions" with two call-to-action
buttons:  "List all students with due date extensions" and "List date
extensions for student"
2) a section called "Resetting extensions" with a single call-to-action
button:  "Reset due date for student"

Agreed that we can build out the docs more here - and perhaps also make the
UI CTA labels more cohesive as well (E.g. use View consistently). I was
thinking these could be V2 polish improvements, after we got a V1 solidly
out...

**Comment by brittneyexline** [comment]
is it possible that this PR is causing failures for the test suite? we have a PR open where the tests are failing, and the same failures happen on master when tests are run locally... this was the last thing that was merged
let me know if there is an easy way to resolve this and we can include it in our PR (https://github.com/edx/edx-documentation/pull/1829)
fyi @saleem-latif

**Comment by stroilova** [comment]
Hi Brittney,

Yes, you might be right, one of the ADA documentation tests might be
failing. We'll explore today.

Thanks,
Olga

**Pull request merged**